tool/jt.rb

```diff
@@ -148,11 +162,12 @@ def help
     puts 'jt irb                                         irb'
     puts 'jt rebuild                                     clean and build'
     puts 'jt run [options] args...                       run JRuby with -X+T and args'
-    puts '    --graal        use Graal (set GRAAL_BIN or it will try to automagically find it)'
-    puts '    --asm          show assembly (implies --graal)'
-    puts '    --server       run an instrumentation server on port 8080'
-    puts '    --igv          make sure IGV is running and dump Graal graphs after partial escape (implies --graal)'
-    puts '    --jdebug       run a JDWP debug server on 8000'
+    puts '    --graal         use Graal (set GRAAL_BIN or it will try to automagically find it)'
+    puts '    --asm           show assembly (implies --graal)'
+    puts '    --server        run an instrumentation server on port 8080'
+    puts '    --igv           make sure IGV is running and dump Graal graphs after partial escape (implies --graal)'
+    puts '    --[j]debug      run a JDWP debug server on 8000'
+    puts '    --jexception[s] print java exceptions'
```

Comment:
FWIW, I like the options here.  The alternative is to raise a hard error on bad options.  I constantly supply `--debug` instead of `--jdebug` and don't realize I've messed up until the script has finished running.  I'd hate to pick the wrong `--jexception` form and think there were no exceptions, when in reality my bad option was just ignored.